source/index.txt

```diff
@@ -1,159 +1,153 @@
-.. _compass-index:
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
 
-===============
-MongoDB Compass
-===============
+========================
+What is MongoDB Compass?
+========================
 
-.. default-domain:: mongodb
+.. introduction::
 
-.. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
-   :class: singlecol
+   MongoDB Compass is a powerful GUI for querying, aggregating, and analyzing your MongoDB data in a visual environment.
 
-What is |compass|?
-------------------
+   Compass is free to use and source available, and can be run on macOS, Windows, and Linux.
 
-|compass| is the :abbr:`GUI (Graphical User Interface)` for MongoDB.
-|compass-short| allows you to analyze and understand the contents of
-your data without formal knowledge of MongoDB
-:manual:`query syntax </tutorial/query-documents>`. In addition to
-exploring your data in a visual environment, you can also use
-|compass-short| to :ref:`optimize query performance <explain-plans>`,
-:ref:`manage indexes <compass-indexes>`, and implement
-:ref:`document validation <validation>`.
+   .. button:: Download Compass
+      :uri: /install
```

Comment:
[question]

I feel like we talked about this in the design review but just wanted to confirm. Is it intentional that both the "Download Compass" button and the "View Installation instructions" both go to the Download and Install doc?

My expected behavior was that the "Download Compass" button would take me right to the download center.

Reply:
@jeff-allen-mongo yes. you're right. The 'Download Compass' should go to the download center (https://www.mongodb.com/try/download/compass) & 'View Installation Instructions' should go to the Install doc.